src/python/grpcio/grpc/experimental/aio/__init__.py

```diff
@@ -45,7 +39,26 @@ def insecure_channel(target, options=None, compression=None):
                    if options is None else options, None, compression)
 
 
+def secure_channel(target, credentials, options=None, compression=None):
+    """Creates a secure asynchronous Channel to a server.
+
+    Args:
+      target: The server address.
+      credentials: A ChannelCredentials instance.
+      options: An optional list of key-value pairs (channel args
+        in gRPC Core runtime) to configure the channel.
+      compression: An optional value indicating the compression method to be
+        used over the lifetime of the channel. This is an EXPERIMENTAL option.
+
+    Returns:
+      An aio (asynchronous) Channel.
```

Comment:
The original docstring is referring to the `Channel` interface, hence the capitalization. How about "An aio.Channel"?